cg/cli/clean.py

```diff
@@ -48,6 +53,48 @@ def beacon(context: click.Context, item_type, item_id):
     api.remove_vars(item_type=item_type, item_id=item_id)
 
 
+@clean.command("balsamic-run-dir")
+@click.option("-y", "--yes", is_flag=True, help="Skip confirmation")
+@click.option("-d", "--dry-run", is_flag=True, help="Shows cases and files that would be cleaned")
+@click.argument("case_id")
+@click.pass_context
+def balsamic_run_dir(context, yes, case_id, dry_run: bool = False):
+    """Remove Balsamic run directory"""
+
+    store = context.obj["db"]
+    balsamic = BalsamicAnalysisAPI(
+        config=context.obj, hk_api=context.obj["hk"], fastq_api=FastqHandler
+    )
+    case_obj = store.family(case_id)
+
+    if case_obj is None:
+        LOG.error("%s: case not found", case_id)
+        context.abort()
+
+    analysis_obj = case_obj.analyses[0] if case_obj.analyses else None
+    if analysis_obj is None:
+        LOG.error("%s: analysis not found", case_id)
+        context.abort()
+
+    analysis_path = balsamic.get_case_path(case_id)
+
+    if yes or click.confirm(f"Do you want to remove {analysis_path}?"):
+
+        if not analysis_path.exists():
+            LOG.warning("could not find: %s", analysis_path)
+            return FAIL
+
+        if analysis_path.is_symlink():
+            LOG.warning("will not automatically delete symlink: %s", analysis_path)
```

Comment:
Is it possible to clarify for the end user what this means or what they should do to follow up?

Reply:
added a message to remove it manually